source/comply/audit-log.rst

```diff
@@ -3,39 +3,205 @@ Audit Log v2 (Experimental) (E20)
 
 *Available in Mattermost Enterprise Edition E20*
 
-System Admins can review a comprehensive listing of events for more in-depth analysis. Additionally, the new audit log provides more control over where the logs are generated and stored.
+System Admins can review a comprehensive listing of events for more in-depth analysis. Additionally, the new audit log provides more control over where the logs are generated and stored. 
+
+The advanced logging capabilities of the Audit Log V2 allow any combination of console, local file, syslog, and TCP socket targets, and send log records to multiple targets. These targets have been chosen as they support the vast majority of log aggregators, and other log analysis tools, without needing additional software installed.
+
+System Admins can define multiple log targets to:
+
+- Mirror log output to files and log aggregators for redundancy.
+- Log certain entries to specific destinations. For example, all errors could be routed to a specific destination for review.
+
+All access to the REST API or CLI is audited. When using Advanced Logging for auditing, System Admins can capture the following auditing in the target configuration in addition to discrete log levels:
+
+.. code-block:: none
+
+   "Levels": [
+      {"ID": 100, "Name": "audit-api"},
+      {"ID": 101, "Name": "audit-content"},
+      {"ID": 102, "Name": "audit-permissions"},
+      {"ID": 103, "Name": "audit-cli"},
+   ],
+
+Where:
+
+- ``audit-api``: Enables output of REST API calls.
+- ``audit-content``: Enables output of API calls that generate content (e.g. ``create post``, ``create reaction``).
+- ``audit-permissions``: Enables output of all permissions failures.
+- ``audit-cli``: Enables output of legacy CLI calls.
+
+All levels can be viewed at ``mattermost-server/shared/mlog/levels.go``.
+
+.. Note::
+  - Logs are recorded asynchronously to reduce latency to the caller. 
+  - Advanced logging supports hot-reloading of logger configuration.
```

Comment:
Hot-reloading of all config files was removed from V6 server, including audit.

Reply:
Will remove this section. Thx for calling out.

Reply:
Actually will add a note for 6.0 for accuracy.